src/clojure_tictactoe/view.clj

```diff
@@ -0,0 +1,33 @@
+(ns clojure-tictactoe.view)
```

Comment:
Do you plan to leave these functions here or in `instructions-printer` and `board-printer`?

Reply:
See above. >.>

Reply:
see above.